### PR TITLE
[#2302] dev: Support to release Doris ci image using pipeline

### PR DESCRIPTION
**Pull request by qqqttt123**
### What changes were proposed in this pull request?

 Support to release Doris ci image using pipeline

### Why are the changes needed?

Fix: #2302 

### Does this PR introduce _any_ user-facing change?
No.

### How was this patch tested?

I will test it after it is merged to master.

### CODE REVIEW

File: .github/workflows/docker-image.yml [hunk 1]
```diff
@@ -39,6 +40,9 @@ jobs:
           elif [ "${{ github.event.inputs.image }}" == "gravitino-ci-trino" ]; then
             echo "image_type=trino" >> $GITHUB_ENV
             echo "image_name=datastrato/gravitino-ci-trino" >> $GITHUB_ENV
+          elif [ "${{ github.event.inputs.image }}" == "gravitino-ci-doris" ]; then
```
Comment (yuqi1129) [inline .github/workflows/docker-image.yml:43]: I'm afraid that you need to consider the architecture here, Doris does not support `all`
image
image

Comment (qqqttt123) [reply .github/workflows/docker-image.yml:43]: Why doesn't support all?

Comment (qqqttt123) [reply .github/workflows/docker-image.yml:43]: Doris should support all type.

Comment (qqqttt123) [reply .github/workflows/docker-image.yml:43]: @zhoukangcn Could you raise a follow-up pr to solve this issue?

Comment (zhoukangcn) [reply .github/workflows/docker-image.yml:43]: Doris has releasing packages for both ARM and x86. You need to separately build the docker image for x86 and ARM.

Comment (qqqttt123) [reply .github/workflows/docker-image.yml:43]: > Doris has different packages for ARM and x86 architectures. You need to separately build the docker image for x86 and ARM.

You can download them to the local machine at the same time. Copy the specific the package in the Docker image.

Comment (zhoukangcn) [reply .github/workflows/docker-image.yml:43]: The package of Doris is too large, with a single core file of BE having 3.8GB. To control the size of the Docker Image, it is recommended to build for ARM and X86 separately.

Comment (qqqttt123) [reply .github/workflows/docker-image.yml:43]: We just download them, we don't copy all of them to the Docker image. we just copy the one which we need.

Comment (zhoukangcn) [reply .github/workflows/docker-image.yml:43]: I guess you mean, we can update `build-docker.sh`, we need build x86 and arm as two step in a single execution when executue with args `--platform=all` and `--type=doris` ?

Comment (zhoukangcn) [reply .github/workflows/docker-image.yml:43]: Now, Doris Image can build as follow：
```
./build-docker.sh --platform linux/arm64 --type doris --image doris_ci --tag test
./build-docker.sh --platform linux/amd64 --type doris --image doris_ci --tag test
```
can you do this in github workflows?

**Comment by qqqttt123** [comment]
After https://github.com/datastrato/gravitino/pull/2327 is merged, we can continue this pr.

**Review by yuqi1129 — APPROVED** [review]
LGTM

**Pull request merged**